dip-0003.md

```diff
@@ -211,17 +211,18 @@ Reorganisations of the chain must also correctly undo changes to the chain-tips
 
 A ProRegTx is invalid if any of these conditions are true:
 
-  1. collateralIndex >= transaction output count
-  2. Any KeyId* field is null (KeyIdOwner, KeyIdOperator or KeyIdVoting)
-  3. KeyIdOwner or KeyIdOperator was already used by any entry in the registered masternodes set
-  4. scriptPayout is not a P2PKH or P2SH script
-  5. When scriptPayout is P2PKH script and the public key hash equals any of KeyIdOwner, KeyIdOperator or KeyIdVoting
-  6. ipAddress is set and port is not set to the default mainnet port
-  7. ipAddress is set and not routable or not an IPv4 mapped address
-  8. ipAddress is set and already used in the registered masternodes set
-  9. operatorReward > 10000
-  10. The inputsHash does not match the calculated hash
-  11. payloadSig is invalid
+  1. collateralOutpoint `hash` is null but an output with 1000 DASH is present at position `n` of the ProRegTx outputs
+  2. collateralOutpoint `hash` is not null but an output with 1000 DASH can't be found in the UTXO specified by the `hash` and `n`
+  3. Any KeyId* field is null (KeyIdOwner, KeyIdOperator or KeyIdVoting)
+  4. KeyIdOwner or KeyIdOperator was already used by any entry in the registered masternodes set
+  5. scriptPayout is not a P2PKH or P2SH script
+  6. When scriptPayout is P2PKH script and the public key hash equals any of KeyIdOwner, KeyIdOperator or KeyIdVoting
+  7. ipAddress is set and port is not set to the default mainnet port
+  8. ipAddress is set and not routable or not an IPv4 mapped address
+  9. ipAddress is set and already used in the registered masternodes set
+  10. operatorReward > 10000
+  11. The inputsHash does not match the calculated hash
+  12. payloadSig is invalid
```

Comment:
```suggestion
  12. In case collateralOutpoint refers to a collateral that is not part of the ProRegTx itself, payloadSig must be a valid signature signed by the collateral public key. If the collateral is part of the ProRegTx, payloadSig must be empty.
```

Reply:
I see what you are trying to say but it doesn't fit into starting `A ProRegTx is invalid if any of these conditions are true:` sentence imo.

Reply:
Updated suggestion to better fit into the list

Reply:
Also added 14., as this only works with P2PKH addresses.